src/Exchange.ts

```diff
@@ -31,30 +25,19 @@ export interface IMatchingOrders {
     gasEstimate: number;
 }
 
-interface IGenericOrder {
+interface IOrder {
     id: number;
     maker: string;
     direction: OrderDirection;
-    bnAmount: BigNumber /** Buy order amount in Wei | Sell order amount in tokens, without decimals */;
-    amount: number /** Buy order amount in ETH | Sell order amount in tokens with decimals */;
-    bnPrice: BigNumber /** price in PPM (parts per million) | price in PPM (parts per million) */;
-    price: number /** price with decimals */;
+    amount: BN /** Buy order amount in Wei | Sell order amount in tokens, without decimals */;
+    price: BN /** price in PPM (parts per million) | price in PPM (parts per million) */;
 }
 
 type IOrderTuple = [string, string, string, string]; /** result from contract: [id, maker, price, amount] */
 
-interface IBuyOrderCalc extends IBuyOrder {
-    bnTokenValue?: BigNumber;
-}
-
-interface ISellOrderCalc extends ISellOrder {
-    bnEthValue?: BigNumber;
-}
-
 interface IExchangeOptions {
```

Comment:
`Augmint.getLegacyExchanges` has to be updated accordingly. (the reason ts didn't  complain because the local `options` var doesn't have the `IExchangeOptions` set which should be fixed too.)